Shared/View/CreateActivityView.swift

```diff
@@ -54,16 +66,33 @@ struct CreateActivityView: View {
             Spacer()
             HStack {
                 Spacer()
-                NavigationLink(destination: VoteView()) {
-                    Text("Create")
+                // Clicking on button will send info to database
+                Button {
+                    Task {
+                        do {
+                            _ = try await ActivityViewModel.createActivity(
```

Comment:
You probably don't want to discard the ActivityID, since you'll need it for the VoteView